docs/diagrams/FindImplementationActivityDiagram.puml

```diff
@@ -10,7 +24,16 @@ if () then ([command is in invalid format])
 
 
 else ([else])
-    : Displays the students that match the search criteria;
+    if () then ([prefix is mt| or lt|])
+'        if () then ([invalid score provided])
+'            :Show invalid value as error message;
+'            stop
+'        else ([else])
+'        endif
+        :find by score <$rake>;
```

Comment:
Great use of the rake symbol. Demonstrates deep technical knowledge of activity diagrams